metals/src/main/scala/scala/meta/internal/implementation/ImplementationProvider.scala

```diff
@@ -320,7 +320,18 @@ final class ImplementationProvider(
         classContext,
         implReal
       )
-      implOccurrence <- findDefOccurrence(implDocument, implSymbol, source)
+      symbolSearch = defaultSymbolSearch(
+        implPath,
+        implDocument
+      )
+      filteredImplSymbol <- Some(implSymbol).filterNot(
```

Comment:
We could just do `if ! isTypeAlias(implSymbol, implDocument) `

Reply:
Oh yeah of course !